api/src/db/migrations/versions/2023_10_18_basic_opportunity_table.py

```diff
@@ -38,11 +38,12 @@ def upgrade():
             nullable=False,
         ),
         sa.PrimaryKeyConstraint("opportunity_id", name=op.f("topportunity_pkey")),
+        schema="api",
```

Comment:
Note that modifying existing database migrations is generally not a great idea / won't do anything. This was necessary to make these changes in order to get our local setup working properly. This should have no effect on anything non-local.